superset-frontend/src/filters/components/TimeColumn/TimeColumnFilterPlugin.tsx

```diff
@@ -24,13 +24,11 @@ import {
   tn,
 } from '@superset-ui/core';
 import React, { useEffect, useState } from 'react';
-import { Select } from 'src/common/components';
+import { Select } from 'src/components';
 import { FormItemProps } from 'antd/lib/form';
-import { Styles, StyledSelect, StyledFormItem, StatusMessage } from '../common';
+import { Styles, StyledFormItem, StatusMessage } from '../common';
```

Comment:
If you agreed with the first one, this `Styles` could also use a more specific/explicit name as well. But then again, these are nits, and I'd love to see this merged!

Reply:
Done